admin/services/data-access/sql.service.js

```diff
@@ -230,32 +241,38 @@ sqlService.transformResult = function (data) {
 sqlService.query = async (sql, params = []) => {
   logger.debug(`sql.service.query(): ${sql}`)
   logger.debug('sql.service.query(): Params ', R.map(R.pick(['name', 'value']), params))
+  await pool
 
   const request = new mssql.Request(pool)
-  if (params) {
-    for (let index = 0; index < params.length; index++) {
-      const param = params[index]
-      // TODO support other options
-      request.input(param.name, param.type, param.value)
+  addParamsToRequestSimple(params, request)
+
+  let result
+  try {
+    result = await request.query(sql)
```

Comment:
Would be good to have a configurable amount of retries and wait times

Reply:
I would quite like to put this minimal version in and see a single recovered query from the logs before committing to a full-featured version.

The load-test environment might be suitable?

Reply:
yep, sounds good